### PR TITLE
fixed description of ellipticity parameter in EllipseGeometry

### PR DESCRIPTION
**Pull request by J-Spitzak**
just fixed a typo that caused me some confusion, also added some additional details about how that parameter is defined

### CODE REVIEW

File: photutils/isophote/geometry.py [hunk 1]
```diff
@@ -85,8 +85,12 @@ class EllipseGeometry:
...
     sma : float
         The semimajor axis of the ellipse in pixels.
-    eps : ellipticity
-        The ellipticity of the ellipse.
+    eps : float
+        The ellipticity of the ellipse. The ellipticity is defined as 
+        .. math::
+
+            \epsilon =  1  -  \frac{b}{a}
+
```
Comment (larrybradley) [inline photutils/isophote/geometry.py:93]: please remove this blank line for consistency within this docstring.

**Comment by J-Spitzak** [comment]
new commit should fix the formatting issues and I added more details about calculating ellipticity

**Review by larrybradley — APPROVED** [review]
Thanks!

**Comment by larrybradley** [comment]
`codestyle` tests is failing because line-too-long (> 100 char):
https://github.com/astropy/photutils/actions/runs/9130160039/job/25108242778?pr=1738#step:6:17

**Comment by larrybradley** [comment]
You can test all the simple style things locally by installing `pre-commit` and running `pre-commit run -a`.

**Pull request merged**